### PR TITLE
Update Cypher type names

### PR DESCRIPTION
**Pull request by injectives**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-drivers-400.surge.sh

File: java-manual/modules/ROOT/pages/cypher-workflow.adoc [hunk 1]
```diff
@@ -144,18 +144,18 @@ The `Value` interface describes methods to infer and coerce the type of values r
...
 // ----
-// include::{java-examples}/ReadingValuesExample.java[tags=java-driver-reading-values-non-null]
+// include::{java-examples}/ReadingValuesExample.java[tags=java-driver-reading-values-non-NULL]
 // ----
 
 // [source, java, indent=0]
 // ----
-// include::{java-examples}/ReadingValuesExample.java[tags=java-driver-reading-values-null]
+// include::{java-examples}/ReadingValuesExample.java[tags=java-driver-reading-values-NULL]
 // ----
```
Comment (Hunterness) [inline java-manual/modules/ROOT/pages/cypher-workflow.adoc]: these for example I'd probably not update to `NULL` but keep it as `null`

File: common-content/modules/ROOT/partials/cypher-workflow.adoc [hunk 2]
```diff
@@ -457,24 +457,24 @@ All types can be potentially found in the result, although not all types can be
...
 |===
 | Cypher Type | Parameter | Result
-| `null`* | {check-mark} | {check-mark}
-| `List`  | {check-mark} | {check-mark}
-| `Map`   | {check-mark} | {check-mark}
-| `Boolean` | {check-mark} | {check-mark}
-| `Integer` | {check-mark} | {check-mark}
-| `Float` | {check-mark} | {check-mark}
-| `String` | {check-mark} | {check-mark}
+| `NULL`* | {check-mark} | {check-mark}
```
Comment (stefano-ottolenghi) [inline common-content/modules/ROOT/partials/cypher-workflow.adoc:460]:
```suggestion
| `null`* | {check-mark} | {check-mark}
```

We should refer to `null` always in the same way. Applies to all instances later below :)

Comment (injectives) [reply common-content/modules/ROOT/partials/cypher-workflow.adoc:460]: I had a chat with @Hunterness and as far as I understand, we should refer to the Cypher type as `NULL`.

Comment (stefano-ottolenghi) [reply common-content/modules/ROOT/partials/cypher-workflow.adoc:460]: > I had a chat with @Hunterness and as far as I understand, we should refer to the Cypher type as NULL.

Fine by me, but why does line 147 make an exception? :thinking:

Comment (injectives) [reply common-content/modules/ROOT/partials/cypher-workflow.adoc:460]: I assume you mean the line `147` mentioned here: https://github.com/neo4j/docs-drivers/pull/400#discussion_r1236792854

It refers to `null` as a value, not the type. @Hunterness please correct me if I am wrong.

**Pull request merged**